README.md

```diff
@@ -1,10 +1,14 @@
 # Project Trieste
 
-Project Trieste is an experimental term rewriting system for experimental programming language development.
+Project Trieste is a term rewriting system designed for rapidly prototyping programming languages.
+Trieste provides three C++ DSLs to enable the rapid prototyping:
 
-This research project is at an early stage and is open sourced to facilitate academic collaborations. We are keen to engage in research collaborations on this project, please do reach out to discuss this.
+* Parsing - Enables generation of an untyped abstract syntax tree (AST) from one or many input files.
+* Rewriting - Enables the restructuring of the AST to simplify, analyse, and elaborate to alternative representations.
```

Comment:
```suggestion
* Rewriting - Enables the restructuring and analysis of the AST. It can be used to simplify, elaborate to alternative representations, and lower to backend IRs or bytecodes.
```